indigo/bulk_creator.py

```diff
@@ -19,15 +19,16 @@
     VocabularyTopic, TaskLabel
 from indigo_api.signals import work_changed
 
+SUBTYPES = Subtype.objects.all()
```

Comment:
Remember, .all() goes to the database. So this will load subtypes once and never pick up any changes. Instead, do this where you need them on line 57.

Reply:
I was trying to avoid going to the db for each row in the spreadsheet given that subtypes are universal and relatively stable. But if a subtype is added and the page is refreshed, we'd definitely want that to be picked up :)

Reply:
Maybe I can get the subtypes once per spreadsheet and pass them along? Do you think that'll be worth the saving?

Reply:
Yes, good idea.